recipes/default.rb

```diff
@@ -556,33 +556,6 @@
   end
 end
 
-# Drop Existing featureStore connection pool and recreate it
-glassfish_asadmin "delete-jdbc-connection-pool --cascade featureStorePool" do
```

Comment:
Shall we keep this to remove the connection pool on upgrades?

Reply:
Yes we should probably do that, otherwise I think the metrics are going to be messed up.